Makefile

```diff
@@ -91,19 +91,19 @@ test:
 	pre-commit run pytest --hook-stage push
 
 # Build a source distribution package and a binary wheel distribution artifact.
-.PHONY: dist bdist-wheel sdist
+.PHONY: dist wheel sdist
 ifeq ($(wildcard .upgraded),)
   PACKAGE_VERSION=unknown
 else
   PACKAGE_VERSION=$(shell python -c 'import package; print(package.__version__)')
 endif
-dist: bdist-wheel sdist
-bdist-wheel: dist/package-$(PACKAGE_VERSION)-py3-none-any.whl
+dist: wheel sdist
+wheel: dist/package-$(PACKAGE_VERSION)-py3-none-any.whl
 dist/package-$(PACKAGE_VERSION)-py3-none-any.whl: check test
-	python setup.py bdist_wheel --dist-dir dist/ --bdist-dir build/
+	flit build --setup-py --format wheel
```

Comment:
Not sure if we would need to set `SOURCE_DATE_EPOCH=$(date +%s)` to avoid non-deterministic time stamps for [reproducible builds](https://flit.pypa.io/en/latest/reproducible.html)? Or is it just effective when we publish using `flit`?

Reply:
My interpretation is that that env variable is for `publish` only 🤔

Reply:
I think we should open a separate issue for reproducible builds.

Reply:
Agreed, and probably with @relekang’s [python-semantic-release](https://github.com/relekang/python-semantic-release) repo, independent of this one.